t/Makefile.am

```diff
@@ -39,7 +39,13 @@ EXTRA_DIST= \
 	expected/test_dbs/small_no_tie.db \
 	expected/test_dbs/small_tie_all.db \
 	expected/test_dbs/small_tie_zero_shares.db \
-	expected/test_dbs/small_tie.db
+	expected/test_dbs/small_tie.db \
+	expected/flux_account/A_bank.expected \
+	expected/flux_account/D_bank.expected \
+	expected/flux_account/full_hierarchy.expected \
+	expected/flux_account/root_bank.expected \
+	expected/pop_db/db_hierarchy_base.expected \
+	expected/pop_db/db_hierarchy_new_users.expected
```

Comment:
You might consider adding a `make distcheck` target to the flux-accounting CI to prevent issues like this slipping through in future PRs.

Reply:
Thanks @grondo! Yes, I completely agree, and I opened an issue on it a little while back (#153), but I just haven't gotten around to it yet. I will try and make it a priority this upcoming month. I'll set MWP